recipe/meta.yaml

```diff
@@ -18,21 +18,21 @@ build:
 
 requirements:
   host:
-    - python >=3.7
+    - python >=3.8
     - pip
     - setuptools
   run:
-    - python >=3.7
+    - python >=3.8
     - cmor >=3.6.0
```

Comment:
@tomvothecoder, this constraint is not listed upstream (https://github.com/E3SM-Project/e3sm_to_cmip/blob/413c37819dd1d7c7674f24f1ff81f61e6b7bdde3/conda-env/ci.yml).  Should I leave it?

Reply:
I see it listed here: https://github.com/E3SM-Project/e3sm_to_cmip/blob/413c37819dd1d7c7674f24f1ff81f61e6b7bdde3/conda-env/ci.yml#L9

Reply:
I'm referring to `cmor >=3.6.0`.  `cmor` is listed without a constraint in that file upstream.

Reply:
The old `conda/meta.yaml` file (which was deleted in this release) had this constraint.  I'm mostly wondering if it got dropped because older versions are now supported or just because it seems unlikely anyone would try to install an older version.

Reply:
Oh I see, I didn't constrain it because I thought it would be unlikely for a user to install an older version. If it ever becomes an issue, I'll add the constraint in the conda env .yml files.

Reply:
As a general rule, I would keep these kinds of constraints.  It presumably indicated that Sterling had encountered some sort of a problem with an older version of `cmor` or knew he was using a newer feature that wasn't available.  But I don't think it's actually likely to come up here because the user base isn't very large and hopefully folks are creating new environments on a regular basis.